### PR TITLE
Fixed retail and wholesale feature based filter issues

### PR DESCRIPTION
**Pull request by mmadariaga**
Check invoices and billing feature properly

#### Type Of Change 
- [X] Small bug fix
- [ ] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [ ] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [X] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

**Review by fmendozzac — APPROVED** [review]

File: web/portal/brand/src/entities/Retail/Form.tsx [hunk 1]
```diff
@@ -47,39 +85,48 @@ const Form = (props: EntityFormProps): JSX.Element => {
...
+          'invoicing.countryName',
+        ],
+      },
     edit && {
       legend: _('Notification options'),
       fields: [
         'voicemailNotificationTemplate',
-        'faxNotificationTemplate',
+        faxEnabled && 'faxNotificationTemplate',
         'invoiceNotificationTemplate',
```
Comment (Kaian) [inline web/portal/brand/src/entities/Retail/Form.tsx:106]: invoiceNotificationTemplate seems like a field that could also be filtered by hasInvoicesFeature, but we can handle that in another PR if you prefer

File: web/portal/brand/src/entities/Wholesale/Form.tsx [hunk 2]
```diff
@@ -18,10 +21,14 @@ const Form = (props: EntityFormProps): JSX.Element => {
...
 
+  const aboutMe = useStoreState((state) => state.clientSession.aboutMe.profile);
+  const hasInvoicesFeature = aboutMe?.features.includes('invoices') || false;
+  const hasBillingFeature = aboutMe?.features.includes('billing') || false;
+
   const groups: Array<FieldsetGroups | false> = [
     {
       legend: _('Basic Configuration'),
-      fields: ['name', 'billingMethod'],
+      fields: ['name', hasBillingFeature && 'billingMethod'],
```
Comment (Kaian) [inline web/portal/brand/src/entities/Wholesale/Form.tsx:31]: I'm ok with this format, as long as linter check passes, but I would have make it multiline when using conditions, seems more clear to me:

```
fields: [
  'name',
  hasBillingFeature && 'billingMethod'
],

**Review by Kaian — APPROVED** [review]

File: web/portal/brand/src/entities/Wholesale/Form.tsx (hunk 2 above)
Comment (mmadariaga) [reply web/portal/brand/src/entities/Wholesale/Form.tsx:31]: it was multiline before the linter changed it :sweat_smile:

File: web/portal/brand/src/entities/Retail/Form.tsx (hunk 1 above)
Comment (mmadariaga) [reply web/portal/brand/src/entities/Retail/Form.tsx:106]: OK, i'll create an issue with that

File: web/portal/brand/src/entities/Wholesale/Form.tsx (hunk 2 above)
Comment (Kaian) [reply web/portal/brand/src/entities/Wholesale/Form.tsx:31]: Doh!

**Pull request merged**